File: totosuki/ABC/B/125.py
N = int(input())
V = list(map(int, input().split()))
C = list(map(int, input().split()))
rslt = 0

for i in range(N):
  if V[i] - C[i] > 0:
    rslt += V[i] - C[i]

print(rslt)

File: OCH_Tao/ABC/B/125.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const N = parseInt(input[0], 10);
const V = (input[1].split(" ")).map(x => parseInt(x, 10));
const C = (input[2].split(" ")).map(x => parseInt(x, 10));
let diff = [];
for (let i = 0; i < N; i++) {
  const X = V[i] - C[i];
  if (X > 0) {
    diff.push(X);
  }
}
console.log(diff.reduce((a, b) => a + b, 0));